cmd/yggd/worker.go

```diff
@@ -192,6 +192,18 @@ func stopWorker(name string) error {
 		return fmt.Errorf("cannot parse data: %w", err)
 	}
 
+	procCmdline, err := os.ReadFile(fmt.Sprintf("/proc/%v/cmdline", pid))
+	if err != nil {
+		return fmt.Errorf("cannot read file /proc/%v/cmdline: %v", pid, err)
+	}
+	if strings.Contains(string(procCmdline), name) {
+		return fmt.Errorf(
```

Comment:
This also should not be fatal error. We should print some warning and delete obsolete pid file.

Reply:
Isn't this condition incorrect too? We should be returning from this function
early if procCmdline does *not* contain the worker name.